### ISSUE TITLE
The 1.6 release thread

### ISSUE THREAD

**Issue by hrydgard** [hrydgard/ppsspp#10773]
I want to get 1.6 out within a reasonable amount of time - the GL threading improvements need to get out there, along with all the other fixes.

Here's the 1.6 milestone tracker: https://github.com/hrydgard/ppsspp/milestone/10

I'm gonna start bumping things from that list onwards to 1.7 soon, except regressions - we shouldn't be worse than 1.5 in any way, preferably. So really, that's what I'm looking for here - has anyone noticed anything that has broken since 1.5.4? If so, please file issues, or let me know here if they are existing ones so I can add them to the 1.6 milestone.

Thanks!

**Comment by leopard20** [comment]
Minus the slower loading and unthrottling issues I didn't notice any other regressions.

Please also consider improving the touch controls, for example adding another "Button Style" and also modifying button magnification on press.
I assume you must've seen the new PS2 emulator that appeared to be using some of PPSSPP codes -DamonPS2- right? That emulator actually had beautiful buttons!

**Comment by leopard20** [comment]
One question though. Can openGl be faster (or maybe more battery efficient) than Vulkan because of the improvements you mentioned?

**Comment by hrydgard** [comment]
I think Vulkan is pretty much always gonna be more power-efficient.. well, unless you're using auto frameskip and the greater performance causes it to actually draw more frames, then the GPU might use more power..

I'm gonna look into the God of War thing soon.

**Comment by chaosenvoys** [comment]
Phantasy star portable 2 problems? Earlier version doesnt have the bugs though..

**Comment by unknownbrackets** [comment]
Bumped a few things that definitely weren't regressions, but not exhaustive.

-[Unknown]

**Comment by hrydgard** [comment]
@chaosenvoys Please link to the issue if you want me to take a look :)

**Comment by chaosenvoys** [comment]
@hrydgard https://github.com/hrydgard/ppsspp/issues/10735
Thabks 😁

**Comment by leopard20** [comment]
One more thing that has changed since v1.5.4 (not a regression though) is that my custom textures don't work anymore ☹️ because they now use new names. It used to work since v1.2 I think.
Thankfully all they need is a simple rename.
Just thought I should mention it in case it was not intended.

**Comment by hrydgard** [comment]
Huh, that's not intentional. Do you have some examples of renames? @unknownbrackets any ideas about that?

**Comment by leopard20** [comment]
For example, these textures are from The 3rd Birthday.
Saved via v1.5.4:
![screenshot_2018-03-25-18-14-39-481_com estrongs android pop](https://user-images.githubusercontent.com/16115820/37875747-aee45afe-3058-11e8-8d04-12f94f1c7479.png)

Saved via v1.5.4-760:
![screenshot_2018-03-25-18-14-26-893_com estrongs android pop](https://user-images.githubusercontent.com/16115820/37875758-d1723dc0-3058-11e8-92fd-c382e146b70b.png)

**Comment by LunaMoo** [comment]
At least one of those looks like a stable, clear/trash-less ui texture and if hash differs on textures like that, it means a hashing method changed.

 I can't reproduce any change on desktop unless I manually switch hashing method, so either those textures do have a part that changed or something with hashing changed on arm/whatever it is.

**Comment by hrydgard** [comment]
Yeah that's not good, we haven't intentionally changed any hashing methods. Please create a new issue and link it from here, with those images and some info about your device like whether it's 32-bit or 64-bit, model, etc.

**Comment by leopard20** [comment]
Please refer to https://github.com/hrydgard/ppsspp/issues/10796 for further discussion on this issue.

**Comment by RaduNastase** [comment]
Android Version. Latest buildbot build

Street Fighter Alpha 3 Max behaves strangely. While the counter shows full speed, the game is strangely choppy and eats a TON of CPU power. The sound is good, so the game is indeed running full speed. I tried many settings, no change to the choppiness.

On 1.2.x and 1.4.x it was running great. 

And sound related.... this happens is pretty much every game i play (DBZ, SFA3, BazBlue, SoulCalibur, Strikers1945, etc) - while all games run full speed, the sound seems to lag sometimes for a brief moment, becoming distorted like a click/pop. Activating auto-frameskip reduces the frequency, but it doesn't eliminate it.  Activating Frameskip though, introduces another small issue in the video department - some minor random stutter.

**Comment by RaduNastase** [comment]
So I extensively tested latest couple of builds in comparison to 1.4.2 on my Galaxy Note 3.

The speed improvements are great. Neo Geo Heroes is now full speed! Soul Calibur is much faster. Dragon Ball Shin Budokai is a bit faster (i'm comparing the CPU usage between builds), BlazBlue is good. Street Fighter Alpha 3 is a mixed bag, but overall is faster i'd say. A little issue would be that ppsspp crashes every time i try to get back into a game after i change rendering resolution.

All games have this annoying issue with Audio though, which was not there previously (or it was much much less noticeable). There are random clicks and pops in all the games, more noticeable in sound effects like punch, dash, charge or special moves. It's random but frequent though and it's definitelly not related to performace as it happens in menus and fmvs as well (the music distorts randomly for a nano second).  This basically the only big issue.

**Comment by Jinzombie** [comment]
The latest build 1.5.4-820 getting slow what happen maybe I should downgrade to 1.5.4-666 that I can play smooth and no crashing like Brave Story during fight scence. This is because you downgrade the CPU consumption in OpenGL?

**Comment by hrydgard** [comment]
@Jinzombie There should be no slowdowns since 672. If there's crashing, you should report it in issues. Everything that's wrong with the very latest builds should be reported!

**Comment by Jinzombie** [comment]
[Screenrecording.zip.zip](https://github.com/hrydgard/ppsspp/files/1882457/Screenrecording.zip.zip)

**Comment by Jinzombie** [comment]
@hrydgard keep crashing in the latest build 1.5.4-835 the game
Brave Story: New Traveler watch my screen recorder above

**Comment by sephiroth0396** [comment]
Is silent hill shattered memories and origins still unplayable? I tried 1.5.4 but it's still unplayable in my device.

**Comment by hrydgard** [comment]
Yeah, they are.

**Comment by sephiroth0396** [comment]
I finished the game on ppsspp honestly but I need to set the ''emulated psp's clock'' to 30 and 60 lmfao

**Comment by hrydgard** [comment]
I plan to do the rollout next weekend. So if anyone got more translations etc to get in, now's the time :)

**Comment by abdellahking** [comment]
Hmmm .great , i'll look into arabic if i see any problme.

**Comment by leopard20** [comment]
@hrydgard 
Are you ok with the name Natural Colors (No Blur) name for the shader I added?! I dunno why it kinda sounds lame to me! :)
Perhaps I should change it to (no AA) or maybe Natural Colors Lite?

**Comment by hrydgard** [comment]
@leopard20  I have no strong opinions on that either way :)

**Comment by leopard20** [comment]
There has to be something you like better!

**Comment by Jinzombie** [comment]
Can't wait for the release fo V1.6.0 😀

**Comment by CYH1996** [comment]
When will it release?

**Comment by marosis** [comment]
@hrydgard please fix dante's inferno. I found you build

**Comment by Sunderland93** [comment]
When will release?

**Comment by iOS4all** [comment]
We hope we get Nintendo switch port of ppsspp when version 1.6 came out.

**Comment by leopard20** [comment]
@iOS4all At the moment there is no Nintendo switch port. You'll have to wait longer than that!

**Comment by iOS4all** [comment]
@lepard20 Thanks for replying.
We really wish to not take so long. Nintendo switch will be a great console to play ppsspp with joycons support it's so much better than touch screen.

Thanks again for your efforts.

**Comment by hrydgard** [comment]
This thread has nothing to do with any possible Switch release. 

As for 1.6, all I can say is very soon, I've been too busy lately...

**Comment by RAKA120598** [comment]
Mr.rydgard,,please make screen keep ON when Loading, because when screen off,,ppsspp will FC or back to home menu,,,,i wish will fix in ppsspp 1.6

**Comment by mrdeathjr28** [comment]
Hi and thanks for your work
Maybe can resolve window resize using vulkan on linux
